nbqa/__main__.py

```diff
@@ -47,6 +48,12 @@
 VIRTUAL_ENVIRONMENTS_URL = (
     "https://realpython.com/python-virtual-environments-a-primer/"
 )
+EXCLUDES = (
+    r"/("
+    r"\.direnv|\.eggs|\.git|\.hg|\.ipynb_checkpoints|\.mypy_cache|\.nox|\.svn|\.tox|\.venv|"
```

Comment:
One way to simplify the regex pattern is to ignore the directories that start with `.`(hidden dirs). check if pattern like `r"/\.(\w|\s|-)+/"`

Reply:
true, but the fact that `black` doesn't do that makes me think we might run into something unexpected if we do that

Reply:
I am wondering why would someone want to run a notebook inside a hidden directory?

Reply:
If someone reports this as a bug/limitation, then we would know the scenario/usecase better right? That would be a better way to know about hidden dirs usage with notebooks.

Reply:
IMHO using an `--exclude` flag/option and glob patterns with [`fnmatch`](https://docs.python.org/3/library/fnmatch.html) as flake8 does, might be more robust and common, than regex.